src/ol/source/imagewmssource.js

```diff
@@ -312,7 +312,7 @@ ol.source.ImageWMS.prototype.getRequestUrl_ = function(extent, size, pixelRatio,
   }
   params['BBOX'] = bbox.join(',');
 
-  return goog.uri.utils.appendParamsFromMap(this.url_, params);
+  return this.url_ + '?' + ol.ext.querystring.stringify(params);
```

Comment:
Mmm yes, good point. Perhaps it would be better to incorporate goog.uri.utils.appendParamsFromMap into ol with the quirk for empty strings removed?